Gemfile

```diff
@@ -136,8 +136,10 @@ gem 'rack-utf8_sanitizer'
 gem 'jwt', '~> 1.5.2', require: false
 
 group :assets do
-  gem 'codemirror-rails', '~> 5.6'
-  gem 'font-awesome-rails', '~> 4.7.0.5'
+  # codemirror-rails is deprecated and won't be updated to be used on Rails 6+
+  # check https://github.com/fixlr/codemirror-rails/issues/61
+  gem 'codemirror-rails', git: 'https://github.com/thalesmiguel/codemirror-rails.git', branch: 'master'
```

Comment:
[This is definitely not cool and needs to be changed]

`codemirror-rails` won't receive more updates, including the ability to run on Rails 6.0 or superior.
To try and run this, I forked the repo (to my personal account) and enabled its use for Rails 6.1.

A couple options to make it better:
- Take this chance to open another PR that removes `codemirror-rails` and, instead, imports whatever's needed for codemirror to work from a JS package;
  - This also means that the ruby code that calls this gem will need to be removed;
- Fork the repo into the `3scale` namespace and perform the same changes I did so we don't need to worry about removing the gem for now.
  - Seems like this was already done in the past with `activejob-uniqueness`.

Reply:
If is the other way around how we did previous updates. We try to update incompatible components while on the old Rails version so we don't have a huge PR at the end.

So if it is feasible to get rid of unsupported gem, that would be preferable in a pull request coming before the rails update one.